src/libraries/Common/src/Interop/Windows/Kernel32/Interop.ReadConsoleInput.cs

```diff
@@ -8,24 +8,26 @@ internal static partial class Interop
 {
     internal const short KEY_EVENT = 1;
 
-    // Windows's KEY_EVENT_RECORD
     [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
-    internal struct KeyEventRecord
+    internal struct KEY_EVENT_RECORD
     {
-        internal BOOL keyDown;
-        internal short repeatCount;
-        internal short virtualKeyCode;
-        internal short virtualScanCode;
-        internal ushort uChar; // Union between WCHAR and ASCII char
-        internal int controlKeyState;
+        internal BOOL bKeyDown;
+        internal short wRepeatCount;
+        internal short wVirtualKeyCode;
+        internal short wVirtualScanCode;
+        private short _uChar; // Union between WCHAR and ASCII char
```

Comment:
```suggestion
        internal ushort wRepeatCount;
        internal ushort wVirtualKeyCode;
        internal ushort wVirtualScanCode;
        private ushort _uChar; // Union between WCHAR and ASCII char
```
to match https://docs.microsoft.com/en-us/windows/console/key-event-record-str